pkger/parser.go

```diff
@@ -668,6 +668,7 @@ func parseChart(r Resource) (chart, []ValidationErr) {
 			Msg:   err.Error(),
 		}}
 	}
+	var failures []ValidationErr
```

Comment:
nit: can you move this close to the spot it is being used? It looks like line 728 is a good spot for it